sdk/src/main/scala/io/horizen/account/state/ForgerStakeStorageV3.scala

```diff
@@ -0,0 +1,404 @@
+package io.horizen.account.state
+
+import com.google.common.primitives.Bytes
+import io.horizen.account.proposition.{AddressProposition, AddressPropositionSerializer}
+import io.horizen.account.state.ForgerStakeStorageV3.ACCOUNT
+import io.horizen.account.state.NativeSmartContractMsgProcessor.NULL_HEX_STRING_32
+import io.horizen.account.utils.BigIntegerUInt256
+import io.horizen.account.utils.WellKnownAddresses.FORGER_STAKE_V2_SMART_CONTRACT_ADDRESS
+import io.horizen.evm.Address
+import io.horizen.proposition.{PublicKey25519Proposition, VrfPublicKey}
+import io.horizen.utils.BytesUtils
+import sparkz.core.serialization.{BytesSerializable, SparkzSerializer}
+import sparkz.crypto.hash.Blake2b256
+import sparkz.util.serialization.{Reader, Writer}
+
+import java.math.BigInteger
+import scala.collection.mutable.ListBuffer
+
+
+object ForgerStakeStorageV3 {
+
+  val ACCOUNT = FORGER_STAKE_V2_SMART_CONTRACT_ADDRESS
```

Comment:
Why do we call this ACCOUNT and not ADDRESS?

Reply:
No particular reason, actually :D 
This is in fact an address, as a type, but the idea was more that this objects lives inside the environment of a specific account and uses its storage. In the end, it's the same, so, if you think it is clearer calling it ADDRESS, I'll change it.